src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -256,7 +256,7 @@ public string ToString(string directory)
         }
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/testing-guidelines/CodeCoverageAnalysis.md.https://codecov.io/gh/PowerShell/PowerShell should work
```powershell
retry of URL failed with error: Connection timed out
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```